subprojects/core/src/main/java/org/gradle/execution/plan/OrdinalGroup.java

```diff
@@ -52,23 +52,4 @@ public boolean isReachableFromEntryPoint() {
     public int getOrdinal() {
         return ordinal;
     }
-
-    @Override
-    public boolean equals(Object o) {
```

Comment:
I think leaving the equals/hashCode in here makes sense, since the type is used as a key in a map.

Reply:
Values are used as map keys but also compared by identity elsewhere so I'd rather make the identity based nature clearer and change the maps to be identity based.